entries/h/homedepot.com.json

```diff
@@ -3,7 +3,9 @@
     "domain": "homedepot.com",
     "url": "https://www.homedepot.com",
```

Comment:
```suggestion
```

Reply:
* Done (in another commit - I should've just done it here)
* I didn't create the Home Depot entry, just using what was already here

Reply:
Thanks!